default/scripting/policies/RACIAL_PURITY.focs.txt

```diff
@@ -9,25 +9,68 @@ Policy
     effectsgroups = [
         [[SPECIES_LIKES_OR_DISLIKES_POLICY_STABILITY_EFFECTS]]
         
+        // This policy disables influence and research foci for non-capital, non-exobot species
+        // See /species/common/focus.macros
+
+        // Creates a special to ensure that deadopting this policy has bad consequencies for a while
+        EffectsGroup
+            scope = Source
+            activation = Not HasSpecial name = "RACIAL_PURITY_SENTINEL_SPECIAL"
+            effects = AddSpecial name = "RACIAL_PURITY_SENTINEL_SPECIAL"
```

Comment:
adds the special every turn while policy is adopted...

Reply:
Should `Not HasSpecial name = "RACIAL_PURITY_SENTINEL_SPECIAL"` go in the scope instead?